docker/demo/compose.simulation.yaml

```
@@ -3,12 +3,17 @@ x-common-config: &common-config
   ipc: host
   environment:
     - RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+    - ROBOT_MODEL=${ROBOT_MODEL:-panther}
```

```suggestion
    - ROBOT_MODEL=${ROBOT_MODEL_NAME:-panther}
```
according to launch file